sdk/vision/azure-ai-vision-imageanalysis/src/test/java/com/azure/ai/vision/imageanalysis/ImageAnalysisClientTestBase.java

```diff
@@ -35,7 +35,7 @@ class ImageAnalysisClientTestBase extends TestProxyTestBase {
     final Boolean printResults = false; // Set to true to print results to console window
 
     // We a single image (the same one) for all error-free tests, one hosted on the web and one local
```

Comment:
nit: seems like a missing word in the comment between `We` and `a`

Reply:
Good catch @joshfree , I fixed the typo. Thank you!